components/firebase_admin_sdk/actions/common/base.mjs

```diff
@@ -0,0 +1,22 @@
+import firebase from "../../firebase_admin_sdk.app.mjs";
+
+export default {
+  props: {
+    firebase,
+  },
+  methods: {
+    async deleteApp() {
```

Comment:
I am wondering if we can remove this function here and call `this.firebase.deleteApp();` directly. Looks like `deleteApp` just call another function